src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -44,7 +43,8 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
-        currLoanList = FXCollections.observableList(new ArrayList<>());
+        filteredLoans = new FilteredList<>(this.addressBook.getLoanList());
+        sortedLoans = new SortedList<>(filteredLoans, Loan::compareTo);
```

Comment:
Making use of the behaviour of `SortedList` is really nice here. Removes the need for implicit sorting through functions at every change. 👍